examples/react-navigation/src/TabNavigator.test.js

```diff
@@ -0,0 +1,17 @@
+import * as React from 'react';
+import { screen, fireEvent } from '@testing-library/react-native';
+import { renderWithNavigation } from './test-utils';
+import TabNavigator from './TabNavigator';
+
+test('Changing tabs', () => {
+  renderWithNavigation(<TabNavigator />);
+  expect(screen.getByRole('header', { name: 'Home screen' })).toBeTruthy();
+
+  // Note: React Navigation uses `button` role for tab buttons as workaround.
+  // It should be actually be `tab` role.
```

Comment:
```suggestion
  // It should actually be `tab` role.
```

Reply:
I'm not sure what you mean when you say it should be `tab` and not `button`, do you mean compared to what's happening on the native side? and why did React Navigation need a workaround?